pkg/loop/internal/pb/keyvalue_store.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+option go_package = "github.com/smartcontractkit/chainlink-common/pkg/loop/internal/pb";
+
+package loop;
+
+import "google/protobuf/empty.proto";
+
+message StoreValueRequest {
+  string key = 1;
+  bytes value = 2;
+}
+
+message GetValueRequest {
+  string key = 1;
+}
+
+message GetValueResponse {
```

Comment:
@ettec Usually these names should mirror the method and add Request/Response, so:
- StoreRequest
- GetRequest
etc.

Reply:
sorted that,  renamed the methods to be a bit more verbose to avoid name clash with other services